site/content/docs/browser-checks/partials-code-snippets.md

```diff
@@ -6,16 +6,19 @@ menu:
     parent: "Browser checks"
 ---
 
-You can reuse commonly used parts of code by referencing code snippets in any browser check. To do this, you use the 
-[Handlebars](https://handlebarsjs.com/guide/partials.html) partial notation `{{> my_code_snippet }}`.
+You can reuse commonly used parts of code by importing code snippets in any Browser check. To do this, you use the standard
+`require(./snippets/my-snippet.js)` syntax supported in the Node.js runtime.
```

Comment:
Why did we specifically say `.js`? I mean, it is okay and js imports support both with and without the extension but was just curious

Reply:
@umutuzgur I want to make it as specific as possible. But both options should work, we just recommend the `.js` extension.

Reply:
Sounds good 👍